src/components/resource/edit-custom-title/resource-edit-custom-title.js

```diff
@@ -312,6 +327,14 @@ export default class ResourceEditCustomTitle extends Component {
                       </label>
                     </Accordion>
 
+                    <CustomLabelsAccordion
+                      editMode
+                      id="resourceShowCustomLabels"
+                      isOpen={sections.resourceShowCustomLabels}
+                      onToggle={this.toggleSection}
+                      userDefinedFields={userDefinedFields}
+                    />
```

Comment:
Shouldn't `customLabels` be passed to `CustomLabelsAccordion`? 
Also, should custom labels accordion be displayed if no custom labels have been created in settings? From what I see now, if there are no custom fields defined, the accordion will be displayed with no content. Should we render the accordion only when custom fields exist?